platform/rct/hal/imu_stm32/imu_device_stm32.c

```diff
@@ -39,7 +39,7 @@
 #define IMU_TASK_PRIORITY	(tskIDLE_PRIORITY + 2)
 #define IS_9150_ADDR            0x68
 
-static struct is9150_all_sensor_data sensor_data[2];
+static struct is9150_all_sensor_data sensor_data[2] = {0};
```

Comment:
I get the premise of the logic but I disagree with the implementation.  Just because you specify initialized data here does not in fact always mean that the compiler will keep this data in the .data section.  IIRC there is a provision in the C standard that says if the code is initialized to all zeros its at the compilers discretion as to whether the code goes to .bss or to .data.  If you need more .bss space then why not adjust memory maps to allocate some more space (I believe you can have more than one .bss section)?  My concern here is that you are making the firmware unnecessarily larger and making it less portable by doing this instead of adjusting memory allocations where necessary.